io.github.chris2511.xca.appdata.xml

```diff
@@ -0,0 +1,92 @@
+            For an easy company-wide use there are customiseable templates that can be used for certificate or request generation.
+        </p>
+        Features
+        <ul>
+            <li>Start your own PKI and create all kinds of private keys, certificates, requests or CRLs</li>
+            <li>Import and export them in any format like PEM, DER, PKCS#7, PKCS#12</li>
+            <li>Use them for your IPsec, OpenVPN, TLS or any other certificate based setup</li>
+            <li>Manage your Smart-Cards via PKCS#11 interface</li>
+            <li>Export certificates and requests as OpenSSL config file</li>
+            <li>Create Subject- and/or Extension- templates to ease issuing similar certs</li>
+            <li>Convert existing certificates or requests to templates</li>
+            <li>Get the broad support of x509v3 extensions as flexible as OpenSSL but user friendlier</li>
+            <li>Adapt the columns to have your important information at a glance</li>
+        </ul>
+    </description>
+    <developer_name>Christian Hohnstädt, Kerstin Steinhauff et al.</developer_name>
+    <releases>
+        <release date="2020-04-29" version="2.3.0">
+            <ul>
+                <li>
+                    Close #191 OID LN differs warning popups at startup
+                </li>
+                <li>
+                    Close #189 Database compaction #189
+                </li>
+                <li>
+                    Improve PKCS11 library loading for portable app
+                </li>
+                <li>
+                    Refactor native separators / and \ on windows.
+                </li>
+                <li>
+                    Support TLS encrypted MariaDB and PostgreSQL connection
+                </li>
+                <li>
+                    Close #182: UI not using Windows native theme in 2.2.1 portable
+                </li>
+                <li>
+                    Close #70: cant open ics file in ical on macos mojave
+                </li>
+                <li>
+                    Close #72: Add checkbox for OCSP staple feature
+                </li>
+                <li>
+                    Use DESTDIR instead of destdir when installing. Follows autotools convention.
+                </li>
+                <li>
+                    Close #172 #46: Multiple OCSP Responders
+                </li>
+                <li>
+                    Close #170 xca-portable-2.2.1 cannot change language
+                </li>
+                <li>
+                    Fix certificate assignment when importing a CA certificate
+                </li>
+                <li>
+                    Close #163: Show key type/size on column of Certificates tab
+                </li>
+            </ul>
+        </release>
+    </releases>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://www.hohnstaedt.de/xca/images/screenshots/XCA-Screenshot-certs.png</image>
```

Comment:
I think each image needs to be in separate `<screenshot>` element.

Reply:
Fixed.